### PR TITLE
Remove NextAuth and Unused Code + Abstracted Storage Logic/Error Handling

### PR DESCRIPTION
**Pull request by kaiyangzheng**
## Description
- Remove NextAuth to migrate to an Azure-specific solution
- Remove unused code and add comments to increase readability
- Added error handling for form blobs not loading

## Motivation and Context
- We want to use Azure's libraries to implement authentication
- Increase readablity of code
- We aren't able to access blob storage currently, so temporarily adding error handling so the web app still works

Closes [#144]

## How has this been tested?

Signed into web app.

## Screenshots (if appropriate):
N/A

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by kaiyangzheng** [comment]
note to update web/.env when this is merged since we no longer need NextAuth and Azure env variables

File: apps/web/src/components/TopBar.tsx [hunk 1]
```diff
@@ -19,18 +19,19 @@ import { useAuth } from '@web/hooks/useAuth';
 import { PositionsService } from '@web/client';
 import { useQuery } from '@tanstack/react-query';
 
+/**
+ * @returns the top bar of the application
+ */
 export const TopBar: React.FC = () => {
-  const { user, signOut } = useAuth();
-  const { data: positions, error: positionsError } = useQuery({
```
Comment (ahweigl) [inline apps/web/src/components/TopBar.tsx:24]: nit: why is `positionsError` not needed anymore?

**Review by ahweigl — APPROVED** [review]
LGTM, thx for adding the jsdocs 👍

File: apps/web/src/components/TopBar.tsx (hunk 1 above)
Comment (kaiyangzheng) [reply apps/web/src/components/TopBar.tsx:24]: It wasn't been used anywhere so I decided to remove it to reduce confusion, we'll add it back if necessary.

**Pull request merged**